app/components/user-search.cjsx

```diff
@@ -14,7 +14,7 @@ module.exports = React.createClass
       apiClient.type('users').get display_name: "#{value}*", page_size: 10
         .then (users) =>
           opts = for user in users
-            { value: user.id, label: "@#{ user.login }: #{ user.display_name }" }
+            { value: user.id, label: "@#{ user.login }" }
```

Comment:
This has still got to have the the display_name since that's what's actually being searched.